python/ray/util/state/state_manager.py

```diff
@@ -307,6 +307,8 @@ async def get_all_task_info(
                 req_filters.job_id = JobID(hex_to_binary(value)).binary()
             elif key == "task_id":
                 req_filters.task_ids.append(TaskID(hex_to_binary(value)).binary())
+            elif key == "is_debugger_paused":
+                req_filters.is_debugger_paused = value == "True"
```

Comment:
oh so we didn't have a `bool` SupportedFilterType here? I actually wonder if this filter needs to be pushed down to GCS if we can just do the filtering in `_filter` in `dashboard/state_aggregator.py` after listing everything from GCS, so we can rely on the boolean filter here: 

https://github.com/ray-project/ray/blob/20f319c2b7642981363093beef3b478c0eb5d052/dashboard/state_aggregator.py#L124-L133

Reply:
Actually, looking through it, this is not needed. I added it when I was debugging why filtering did not work but I removed it now and it still works.

This is now removed.